L1Trigger/L1THGCal/interface/concentrator/HGCalConcentratorSuperTriggerCellImpl.h

```diff
@@ -23,15 +26,15 @@ class HGCalConcentratorSuperTriggerCellImpl
   private:
 
     int getSuperTriggerCellId(int detid) const ;
-    static const int kSplit12_ = 0x3a;
-    static const int kSplit3_ = 0x30;
+    static std::map<int,int> kSplit_;
     static const int kWafer_offset_ = 6;
-
     static const int kSTCsize16_ = 16;
     static const int kSTCsize4_ = 4;
     static const int kNLayers_ = 3;
+    static const int kSplit12_v9 = 0x36;
```

Comment:
Again a very minor comment, but since there is only  one `kSplit`, the name could be simplified to `kSplit_v9_` (also please add the trailing underscore)